lib/peanut_labs/builder/iframe_url.rb

```diff
@@ -20,10 +16,10 @@ def initialize(params=nil)
 #     params[:dob] -> not required, classes accepted - Date, DateTime, Time or formatted "MM-DD-YYYY" string
 #     params[:sex] -> not required, 1 for male, 2 for female
 #
-      def call(params)
+      def self.call(params)
         raise PeanutLabs::UserIdMissingError if params[:id].nil? || params[:id].empty?
```

Comment:
Sorry, I hate Active support with passion. Don't really want to bring it just for this case.